src/index.ts

```diff
@@ -120,6 +159,12 @@ async function main() {
     polkaBtcApi.redeem.setAccount(account);
 
     // await floodFaucet(polkaBtcApi, 100);
-    await callIssueAndRedeem(polkaBtcApi, account);
-    setInterval(callIssueAndRedeem, requestWaitingTime, polkaBtcApi, account);
+    if (REDEEM_EXECUTION_MODE) {
```

Comment:
For a future PR, what do you think if we restructure this to work similarly to the testdata-gen tool? Such that you would pass the action that you execute as a CLI argument instead?